content/hi/docs/reference/glossary/cluster.md

```diff
@@ -0,0 +1,20 @@
+---
+title: क्लस्टर
+id: cluster
+date: 2019-06-15
+full_link: 
+short_description: >
+  वर्कर मशीनों का एक समूह, जिसे नोड्स भी कहा जाता है, जो कंटेनरीकृत एप्लिकेशन्स चलाते हैं। हर क्लस्टर में कम से कम एक वर्कर नोड होता है।
+
+aka: 
+tags:
+- fundamental
+- operation
+---
+वर्कर मशीनों का एक समूह, जिसे {{< glossary_tooltip text="नोड्स" term_id="node" >}} भी कहा जाता है,
+जो कंटेनरीकृत एप्लिकेशन्स चलाते हैं। हर क्लस्टर में कम से कम एक वर्कर नोड होता है।
+
+<!--more-->
+वर्कर मशीन {{< glossary_tooltip text="पॉड्स" term_id="pod" >}} को होस्ट करते है जो कि एप्लिकेशन वर्कलोड के घटक हैं।  
+{{< glossary_tooltip text="कंट्रोल प्लेन" term_id="control-plane" >}} क्लस्टर में वर्कर नोड्स और पॉड्स का प्रबंधन करता है। एक प्रोडक्शन  वातावरण में, कंट्रोल प्लेन आमतौर पर
```

Comment:
A small nit but otherwise LGTM
```suggestion
{{< glossary_tooltip text="कंट्रोल प्लेन" term_id="control-plane" >}} क्लस्टर में वर्कर नोड्स और पॉड्स का प्रबंधन करता है। एक प्रोडक्शन वातावरण में, कंट्रोल प्लेन आमतौर पर
```